README.md

```diff
@@ -3866,51 +3866,119 @@ _You can enable the following settings in Xcode by running [this script](resourc
 * <a id='subsection-organization'></a>(<a href='#subsection-organization'>link</a>) **Within each top-level section, place content in the following order.** This allows a new reader of your code to more easily find what they are looking for. [![SwiftFormat: organizeDeclarations](https://img.shields.io/badge/SwiftFormat-organizeDeclarations-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#organizeDeclarations)
   * Nested types and type aliases
   * Static properties
-  * Class properties
+  * Static property with body
+  * Class properties with body
+  * SwiftUI dynamic properties (@State, @Environment, @Binding, etc)
```

Comment:
@miguel-jimenez-0529 what do you think about generalizing this section to be any property with a property wrapper? I'm curious about the reason to scope this specifically to SwiftUI property wrappers. cc: @calda

Reply:
Me and Cal had the same conversation. Our conclusion was that there are `@` properties like `@objc` that are not property wrappers. Also a property wrapper doesn't have a common semantic meaning, examples of those are `@Referenced`, `@UniBindable`, `@Atomic`, `@Resilient`. So grouping all property wrappers when we don't know upfront their functionality was a bit opinionated. Engs may have a better idea on where to place them.
This is different for SwiftUI property wrappers since they all conform to the `DynamicProperty` protocol and have a common semantic meaning in the sense that they interact with the UI.

Reply:
This makes sense. Thanks for explaining!